src/roman_datamodels/stnode/_node.py

```diff
@@ -29,9 +25,6 @@ class DNode(MutableMapping):
 
     __slots__ = ("_data", "_name", "_parent", "_read_tag")
 
-    _pattern: ClassVar[str]
-    _latest_manifest: ClassVar[str]
```

Comment:
These were only useful to the tagged nodes so I put them there, note that `_read_tag` will be used here after we do the tagged scalars, so I didn't move it.